modules/ocf_tv/files/i3/config

```diff
@@ -29,13 +34,13 @@ font pango:DejaVu Sans Mono 8
 floating_modifier $mod
 
 # start a terminal
-bindsym $mod+Return exec i3-sensible-terminal
+bindsym $mod+Return exec $term
 
 # kill focused window
 bindsym $mod+Shift+q kill
 
-# start dmenu (a program launcher)
-bindsym $mod+d exec dmenu_run
+# start rofi (a program launcher)
+bindsym $mod+d exec rofi -show run
```

Comment:
If `drun` is available we should probably use it, since that gives more sensible program names.

Reply:
Yeah, it is. Good idea.